packages/phone-number-privacy/signer/src/server.ts

```diff
@@ -98,7 +98,8 @@ export function startSigner(
       new PnpQuotaIO(
         config.api.phoneNumberPrivacy.enabled,
         config.api.phoneNumberPrivacy.shouldFailOpen, // TODO (https://github.com/celo-org/celo-monorepo/issues/9862) consider refactoring config to make the code cleaner
-        kit
+        kit,
+        config.timeout
```

Comment:
So this timeout is for the signer's overall processing of requests. If a request takes longer than this to process, the signer stops processing it. 

We should add a new timeout value to the SignerConfig for full node requests. If you add this to the BlockchainConfig type (as I suggested in my comments on the combiner changes) you'll inherit that into SignerConfig automatically. You could then access the timeout like `config.blockchain.timeoutMs`